### ISSUE TITLE
Vimeo Video Not Working/Stopped Loading

### ISSUE THREAD

**Issue by masoodanwar-jin** [artpolikarpov/fotorama#565]
Hello,
We are using the Fotorama slider for displaying Vimeo videos on our website but it suddenly stopped working. The Vimeo videos are not loading anymore, we followed the documentation but the videos are not loading on https://fotorama.io/docs/4/video/ as well. As can be seen, the YouTube videos are working fine but the Vimeo videos are not working now, please check the "Vimeo Staff Pick" section, it is not working on the examples page https://fotorama.io/examples/video/ as well.

### Preconditions

- The Vimeo videos were loading and working fine.

### Steps to reproduce

- Please visit https://fotorama.io/docs/4/video or https://fotorama.io/examples/video
- Check the Vimeo videos in the slider.

### Expected result
The Vimeo video should load and be playable.

### Actual result
The video is not showing/loading and is not playing anymore.

Please check what is the issue so we can fix it on our side.

Thanks

![fotorama-io-video-issue-1](https://user-images.githubusercontent.com/77829367/105467684-d9eeab80-5cb7-11eb-9e33-2b2ca1a0adcc.png)

![fotorama-io-video-issue-2](https://user-images.githubusercontent.com/77829367/105467697-dc510580-5cb7-11eb-8a2a-70fc13551857.png)

**Comment by HMadhanguru** [comment]
Hi @artpolikarpov , I'm having the same issue on my site. May I know if this issue will be fixed anytime soon?

**Comment by mmoffatt-nzaa** [comment]
I've just encountered this problem on a client site too. Has something changed at Vimeo's end that has broken this?

**Comment by masoodanwar-jin** [comment]
Yes, it is on Vimeo's end and other software had already added handling for the changes and fixed the issue, but no updates for @artpolikarpov fotorama yet.

**Comment by TobiasHenzeOSF** [comment]
> Yes, it is on Vimeo's end and other software had already added handling for the changes and fixed the issue, but no updates for @artpolikarpov fotorama yet.

can you name an alternative please? @masoodanwar-jin

**Comment by babsevensix** [comment]
I also have the same problem
Has anyone found any solution?
